src/test/java/com/microsoft/alm/gitcredentialmanager/ProgramTest.java

```diff
@@ -110,13 +114,13 @@
 
     @Test public void checkOsRequirements_macOsBadVersion()
     {
-        Assert.assertEquals(1, Program.checkOsRequirements("Mac OS X", "10.10.1").size());
+        Assert.assertEquals(1, Program.checkOsRequirements("Mac OS X", "10.9.1").size());

```

Comment:
This one duplicates the test on line 125, so you could simply delete it and the blank line below.